### PR TITLE
Integrate new firebase implementation

### PR DESCRIPTION
**Pull request by mmehmetAliIzci**
Deleted all unnecessary implementations thanks to Native ALM 👍

### CODE REVIEW

File: content/howto/mobile/setting-up-native-push-notifications.md [hunk 1]
```diff
@@ -187,64 +96,21 @@ Remote notifications will not work with iOS Simulator.
...
+In **Signing & Capabilities** tab, add **push notifications** capabilities to your app by doing the following:
+1. Click **nativeTemplate** in the left-hand file explorer.
+2. Click the **Signing & Capabilities** tab.
+3. Click **Targets** > **nativeTemplate**.
+4. Click **Capability**, type *push* in the dialog box, then select **Push Notifications**.
+5. Click **Capability**, type *background* in the dialog box, then select **Background Modes**.
+6. Select **Background Modes** > **Remote Notifications**:
 
-     ![Capabilities](attachments/native-remote-push/iosCustomizations.png)
+![Capabilities](attachments/native-remote-push/iosCustomizations.png)
```
Comment (ConnorLand) [inline content/howto/mobile/setting-up-native-push-notifications.md]: @mmehmetAliIzci can you still get to this screen easily? I want to increase the number size and add a 6.

**Comment by ConnorLand** [comment]
@mmehmetAliIzci how is this going to affect PR #2132 ?

File: content/howto/mobile/setting-up-native-push-notifications.md (hunk 1 above)
Comment (mmehmetAliIzci) [reply content/howto/mobile/setting-up-native-push-notifications.md]: Yeah its pretty straight forward in my opinion to whomever knows arithmetics and see the numbers :)

Comment (ConnorLand) [reply content/howto/mobile/setting-up-native-push-notifications.md]: Right, but I think you have 6 steps and only 5 numbers on the picture.

Comment (ConnorLand) [reply content/howto/mobile/setting-up-native-push-notifications.md]: @mmehmetAliIzci is it possible for you to add a number 6 to the "capabilities" picture aligned with the new steps I've proofed around it? After that I will merge this PR into the main PR.

**Pull request merged**